liblustreapi-sys/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2019 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+#![allow(non_upper_case_globals)]
+#![allow(non_camel_case_types)]
+#![allow(non_snake_case)]
+
+include!("bindings.rs");
+
+type lstat_t = libc::stat64;
```

Comment:
I'm seeing an error here that `stat64` doesn't exist.

Should this be `libc::stat`?

Reply:
Must be because I'm building on mac